engine/class_modules/sc_hunter.cpp

```diff
@@ -3421,6 +3472,9 @@ struct aimed_shot_base_t: public hunter_ranged_attack_t
     base_aoe_multiplier = p -> specs.trick_shots -> effectN( 4 ).percent() +
                           p -> conduits.deadly_chain.percent();
 
+    if ( p -> options.t28_2pc )
+      base_aoe_multiplier += 0.3;
```

Comment:
According to spell data 2pc adds a damage increase on the Trick Shots buff - https://ptr.wowhead.com/spell=257622/trick-shots the second effect is new in 9.2 and while it's 0 in the spell data, experience tells me that 2pc simply updates this value. Which means it increases *all* AiS/RF damage while it's up and is not passive.

Reply:
What should I change for this?

Reply:
Handle it inside `composite_da_multiplier()` in `rapid_fire_t::damage_t` (like the hidden BP buff) and `aimed_shot_base_t` (have to add that and Tricks should be checked with `trick_shots_up()` because of the interaction with DT, will need to test that once PTR is up though in case they played with TrS while I was away).

Reply:
Updated, please check it's implemented correct

Reply:
Ok for RF. For AiS move it to `composite_da_multiplier()` please, it's a buff so it's not target specific.